packages/thriver-core-components/lib/component-tabs/tabs.js

```diff
@@ -47,6 +47,25 @@ Template.body.events({
  * @method
  */
 Template.tabs.onRendered(function () {
+
+    //Deselect all active tabs if on mobile
+    if (window.innerWidth < 768){
+
+        // Tabs vars
+        var tabAnchors = document.querySelectorAll('menu.tabs > li > a'),
+            tabs       = document.querySelectorAll('div.tabs > article');
+
+        // Remove expanded from all tab anchors
+        for (let i = 0; i < tabAnchors.length; ++i)
+            Thriver.util.makeActive(tabAnchors[i], false);
+
+        // Add hidden state to all tabs
+        for (let i = 0; i < tabs.length; ++i)
```

Comment:
Loops and conditionals must use curly braces unless their only statement exists on the same line, per the airbnb style guide.